pulsar-client-kafka-compat/pulsar-client-kafka/src/test/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumerTest.java

```diff
@@ -94,6 +120,10 @@ public void testPulsarKafkaConsumerWithHeaders() throws Exception {
         pulsarKafkaConsumer.received(consumer, msg);
         pulsarKafkaConsumer.poll(100);
         pulsarKafkaConsumer.close();
+
+
+        Assert.assertNotNull(msg.getProperty(kafkaHeader));
```

Comment:
Can we assert the value of the header ?

Reply:
Asserted Header value from message property